how_to_how_to.md

```diff
@@ -0,0 +1,69 @@
+# How to write a NumPy how-to
+
+How-tos get straight to the point. They either answer a focused question, or narrow down
+broad questions into a set of focused questions that the user can then choose among.
+
+## A stranger has asked for directions
+
+*"I need to fill my tank."*
+
+## Brief but explicit answer
+
+  * Three miles, take a right at Hayseed Road, it's on your left.
+
+Add useful details ("Hayseed Road", even though it's the only turnoff at three miles), but not irrelevant ones. Don't say "A lot of people have trouble finding fuel in this town." If you're on Route 3, don't also explain how to get there from Route 7. If there's interesting background (tutorial, explanation, reference, alternative approach), link to it so the user can discover it, but that information doesn't belong in a how-to.
+
+## Delegate
+
+  * Three miles, take a right at Hayseed Road, follow the signs.
+
+If the information is already in the docs, link to it -- don't repeat it. If the API doc is sufficiently clear, link directly to it after you've given the prefatory information ("Three miles, take a right").

```

Comment:
nit: I had no idea what prefatory meant (loved learning about it!), maybe change it to a simpler word for the benefit of non-native English speakers?

Reply:
I really like this How-to guide for how-tos @bjnath - thank you!

As @melissawm said, maybe change it to a simpler word. What do you think about using "introductory"? (Ref: https://developers.google.com/style/translation#be-inclusive)

Cheers!